front-end/app/add-side-modal/add-side-modal.component.ts

```diff
@@ -0,0 +1,33 @@
+import { Component } from '@angular/core';
```

Comment:
What made you decide to call this the AddSideModal? Is it something you are likely to use for customising other sides or is it quite specific to just this one item?